volatility3/framework/plugins/windows/info.py

```diff
@@ -166,6 +257,7 @@ def _generator(self):
         for i, layer in self.get_depends(self.context, layer_name):
             yield (0, (layer.name, f"{i} {layer.__class__.__name__}"))
 
+        yield (0, ("Is KDBG encoded: ", str(bool(self.is_kdbg_encoded(self.context, layer_name, symbol_table)))))
```

Comment:
Because we put a delimiter in the rendering process (using a line or blank if we use Pretty), I think we don't need to put a separate delimiter (`:`) when we yield data.

```suggestion
        yield (0, ("Is KDBG Encoded", str(bool(self.is_kdbg_encoded(self.context, layer_name, symbol_table)))))
```

Reply:
replaced it